content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -71,7 +71,7 @@ Data plane performance depends on many factors, for example:
 - Number of proxy worker threads
 - Protocol
 - CPU cores
-- Number and types of proxy filters, specifically telemetry v2 related filters.
+- Number and types of proxy filters, specifically telemetry related filters.
```

Comment:
Can a user customise these?  Should we say something like "Number and type of Envoy filters, including those added by telemetry features" ?